config/Dockerfiles/fedora29/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN dnf -y install python-pip ansible curl gcc python-devel python2-shade \
                    libvirt-daemon-kvm qemu-kvm libvirt-daemon-config-network \
                    libvirt-python libvirt-devel redhat-rpm-config file \
                    openssh genisoimage libvirt-client virt-install net-tools \
-                   python-krbV make libxslt-python krb5-workstation jq buildah \
+                   python-krbV make libxslt-python krb5-workstation jq buildah git \
```

Comment:
I don't think you need git, if the slaves did not have git package they wouldn't be able to clone, so I'd suggest dropping here and checking as well. It could be that it needs dependencies and we are failing to catch the stderr

Reply:
These aren't jenkins slaves.

Reply:
I see, these are just for distros independently, thanks!